app/views/relationships/create.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#follow_form').html('<%= escape_javascript(render('users/unfollow')) %>');
+$('#followers').html('<%= @user.followers.count %>');
```

Comment:
1. Use single quote in js file
2. Use double quote in ruby
3. Use j instead of  escape_javascript
4. Remove redundant (), in summary
```ruby
$('#follow_form').html('<%= j render "users/unfollow" %>');
```